### PR TITLE
[FIX] delivery_schenker: Bugfix volume calculation

### PR DESCRIPTION
**Pull request by mt-software-de**
Sorry there were two bugs in the code from https://github.com/OCA/delivery-carrier/pull/622. 

I think we should add a simple test for it, :-) I am trying to do this today. 

cc @pedrobaeza @hildickethan-S73

### CODE REVIEW

File: delivery_schenker_picking_volume/models/delivery_carrier.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 class DeliveryCarrier(models.Model):
     _inherit = "delivery.carrier"
 
-    def _schenker_shipping_information(self, picking):
-        res = super()._schenker_shipping_information(picking)
-        res["volume"] = picking.volume
-        return res
+    def _prepare_schenker_shipping(self, picking):
+        vals = super()._schenker_shipping_information(picking)
```
Comment (hildickethan) [inline delivery_schenker_picking_volume/models/delivery_carrier.py]:
```suggestion
    def _prepare_schenker_shipping(self, picking):
        vals = super()._prepare_schenker_shipping(picking)
```

File: delivery_schenker_picking_volume/models/delivery_carrier.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
+        vals["volume"] = picking.volume
```
Comment (hildickethan) [inline delivery_schenker_picking_volume/models/delivery_carrier.py]:
```suggestion
        vals["shippingInformation"]["volume"] = picking.volume
```

Comment (mt-software-de) [reply delivery_schenker_picking_volume/models/delivery_carrier.py]: thanks

File: delivery_schenker_picking_volume/models/delivery_carrier.py (hunk 1 above)
Comment (mt-software-de) [reply delivery_schenker_picking_volume/models/delivery_carrier.py]: :-) thanks

**Review by hildickethan — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**